src/omero_zarr/raw_pixels.py

```diff
@@ -275,14 +328,20 @@ def plate_to_zarr(plate: omero.gateway._PlateWrapper, args: argparse.Namespace)
 
 def add_multiscales_metadata(
```

Comment:
Suggested in https://github.com/ome/ome-zarr-py/pull/124#discussion_r783818649, I would propose we use the opportunity of the upcoming API changes associated with the 0.4 support to remove the duplication in our code base. In this case, I expect we should be able to start consuming `ome_zarr.writer.write_multiscales_metada` directly.

Reply:
Done in 70975a4